docs/synapse/userguides/storm_ref_filter.rstorm

```diff
@@ -931,3 +932,55 @@ Refer to the :ref:`storm-ref-subquery` guide for additional information on subqu
 ::
 
     <file:bytes> +{ -> it:av:filehit }>=10
+
+
+.. _double_colon_syntax:
+
+Double Colon Syntax
+---------------------
```

Comment:
We have previously (internally) called this "implicit pivot" syntax. Ideally i'd like to coin a term for what id does rather than the nature of the syntax if we can :) . Also need to trim 2 `--`s i think